src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
+    private int area;
```

Comment:
Consider changing the type of the `area` field from `int` to `double` to match the return type of the `getArea` method and to accommodate non-integer area values.